src/tests/tracing/eventpipe/common/IpcUtils.cs

```diff
@@ -446,6 +446,12 @@ public static Stream GetStandardTransport(int processId)
                 namedPipe.Connect(3);
                 return namedPipe;
             }
+            else if (OperatingSystem.IsAndroid())
+            {
+                Int32 port = 9000;
+                TcpClient client = new TcpClient("127.0.0.1", port);
```

Comment:
Android emulator doesn't use 127.0.0.1 as its IP (unless you have setup port forwarding using adb). Is that the case? If not, then emulator uses 10.0.2.2 as its loopback interface IP address.

Reply:
I has tried using 10.0.2.2 here, but `System.Net.Internals.SocketExceptionFactory+ExtendedSocketException (111): Connection refused 10.0.2.2:9000` is hit. I was guessing that this needs to be the same value as `DiagnosticsPort` set in `build.proj`, and my initial attempts to use `10.0.2.2` didn't work there either.
Should `processinfo.cs` fail here when using `GetStandardTransport`?

Reply:
We run the client directly on the device for this scenario, so it will be good.